test/server/cards/01_SOR/leaders/CassianAndorDedicatedToTheRebellion.spec.ts

```diff
@@ -0,0 +1,95 @@
+describe('Cassian Andor, Dedicated to the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Cassian Andor\'s leader ability', function() {
+            beforeEach(function() {
+                contextRef.setupTest({
```

Comment:
If there's only one test case for a given setupTest then you can put the setup directly in the test case instead of doing a beforeEach. See e.g. the Palpatine leader tests for an example.

Reply:
Cleaned this up